### PR TITLE
Updates kdp to be minikube friendly

### PR DESCRIPTION
**Pull request by LtChae**
co-authored-by: Nathaniel Stevens <nstevens@pillartechnology.com>

Keep in mind that this assumes a fairly robust machine running minikube.

### CODE REVIEW

File: kubernetes-data-platform/values.yaml [hunk 1]
```diff
@@ -34,8 +34,10 @@ presto:
       rollingUpdate:
         maxSurge: 1
         maxUnavailable: 1
+    deployPrometheusExporter: false
+    useJmxExporter: false
   ingress:
-    enable: true
+    enable: false
```
Comment (jeffgrunewald) [inline kubernetes-data-platform/values.yaml]: if we're targeting local development with our "sane defaults" does it make more sense to have this set to true and let the users turn it off in prod for security reasons?

File: kubernetes-data-platform/values.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 presto:
-  workers: 2
+  workers: 0
```
Comment (jeffgrunewald) [inline kubernetes-data-platform/values.yaml]: ambivalent here; technically presto can use the coordinator as a worker for something like local development but i believe there's a flag you have to set in the presto config to say "assign workloads to the coordinator" if you're going that route; alternatively i could make the argument of having a single worker by default to more closely mirror a "prod-like" scenario.

File: kubernetes-data-platform/values.yaml (hunk 1 above)
Comment (LtChae) [reply kubernetes-data-platform/values.yaml]: Can minikube do ingresses? Where do we get the needed annotations?

File: kubernetes-data-platform/values.yaml (hunk 2 above)
Comment (LtChae) [reply kubernetes-data-platform/values.yaml]: We set it up this way to minimize the amount of system resources the cluster consumes. It's already heavy as it is.

File: kubernetes-data-platform/values.yaml (hunk 1 above)
Comment (jeffgrunewald) [reply kubernetes-data-platform/values.yaml]: yes; we will need to default to running one of the other non-AWS specific ingress controllers and then flip over to the one specific to your cloud provider via chart deploy-time config. i will create a separate card for that one.

File: kubernetes-data-platform/values.yaml (hunk 2 above)
Comment (jeffgrunewald) [reply kubernetes-data-platform/values.yaml]: that's why i was on the fence; don't feel strongly either way.

**Pull request merged**